safe-room.lic

```diff
@@ -47,6 +57,14 @@ class SafeRoom
     give_and_take(settings.safe_room_id, settings.safe_room_give, settings.safe_room_take)
   end
 
+  def get_health_score(health_hash)
```

Comment:
I normally avoid one-liners since they reduce readability, but can get it switched out if that's how you'd like to see it